.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -1173,7 +1173,7 @@ tsh bench web sessions --max=5000 --web user ls
   - [ ] `desktop.directory.share` (`TDP04I`) emitted when Teleport starts sharing a directory
   - [ ] `desktop.directory.read` (`TDP05I`) emitted when a file is read over the shared directory
   - [ ] `desktop.directory.write` (`TDP06I`) emitted when a file is written to over the shared directory
-- Warnings/Errors
+- Warnings/Errors (test by applying [this patch](https://gist.github.com/ibeckermayer/7591333275e87ad0d7afa028a7bb54cb))
```

Comment:
This is alright, but is there an organic way to test this? Alternatively, is this test worth having in testplan form, given that it needs a code patch to work?

Reply:
The short answer is that yes it's possible, but it's difficult to induce these organically, hence this approach.